eks-addons.tf

```diff
@@ -0,0 +1,23 @@
+module "eks_addons" {
+  source               = "./modules/eks_addons"
+  enable_ingress_nginx = var.ingress_nginx_config.enable
```

Comment:
Since this value is set in ingress_nginx_config it does not make sense to have 2 times same value sent to module. It could maybe better just to reuse already exsting value which is set to ingress_nginx_helm_config.

Reply:
It is not actually set in the ingress_nginx_helm_config variable, since it is not part "helm_config".
We could move it into it, but then i would like to change "ingress_nginx_helm_config" into "ingress_nginx_config". If we do it this way i would also like to simplify this entirely.

- Input variable for module then should variable from root variables, extended with "subnet_ids" list variable.
- nginx_values.yaml then should be moved to modules/eks_addons/values_templates/nginx_values.yaml
- values gets composed within ingress-nginx.tf, instead of eks_addons

Reply:
Changes added.